### PR TITLE
[AMORO-3981][FOLLOWUP] Refine the duration to string format 

### PR DESCRIPTION
**Pull request by turboFei**
## Why are the changes needed?

Close #xxx.
In https://github.com/apache/amoro/pull/3982, I support to generate the conf docs automatically.

However, the default value looks not human readable.

All the duration default value is in ns time unit.
image

In this PR, we refine the Duration format to string.

## Brief change log

- Refine the duration format to string

## How was this patch tested?

- [x] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [] Add screenshots for manual tests if appropriate

- [x] Run test locally before making a pull request

## Documentation

- Does this pull request introduce a new feature? (yes / no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

**Comment by turboFei** [comment]
I will regenerate the docs after #3982 got merged.

**Comment by turboFei** [comment]
cc @huyuanfeng2018

**Review by huyuanfeng2018 — COMMENTED** [review]
I think need to refer to it here: `org.apache.amoro.config.ConfigHelpers#TimeUtils`
For example, using `1 min` display on the page is better than using `PT1M`

**Comment by turboFei** [comment]
> I think need to refer to it here: `org.apache.amoro.config.ConfigHelpers#TimeUtils`

Thanks for the suggestion, I have updated.

File: docs/configuration/ams-config.md [hunk 1]
```diff
@@ -52,10 +52,10 @@ table td:last-child, table th:last-child { width: 40%; word-break: break-all; }
 | clean-dangling-delete-files.enabled | true | Enable dangling delete files cleaning. |
 | clean-dangling-delete-files.thread-count | 10 | The number of threads used for dangling delete files cleaning. |
 | clean-orphan-files.enabled | true | Enable orphan files cleaning. |
-| clean-orphan-files.interval | 86400000 ms | Interval for cleaning orphan files. |
+| clean-orphan-files.interval | 1 d | Interval for cleaning orphan files. |
```
Comment (turboFei) [inline docs/configuration/ams-config.md:55]:
```
              // Use the largest unit if all divide evenly
              .orElse(orderedUnits.get(orderedUnits.size() - 1));
```

It is used to show `1 d` instead of `86400000 ms`.
